eng/docker/build-docker-sdk.ps1

```diff
@@ -34,12 +29,35 @@ if ($buildWindowsContainers)
   }
 
   $dockerFile="$dockerFilePrefix.windows.Dockerfile"
+  
+  # Collect the following artifacts to folder, that will be used as build context for the container,
+  # so projects can build and test against the live-built runtime:
+  # 1. Reference assembly pack (microsoft.netcore.app.ref)
+  # 2. Runtime pack (microsoft.netcore.app.runtime.linux-x64)
+  # 3. targetingpacks.targets, so stress test builds can target the live-built runtime instead of the one in the pre-installed SDK
+  # 4. testhost
+  $binArtifacts = "$REPO_ROOT_DIR\artifacts\bin"
+  $dockerContext = "$REPO_ROOT_DIR\artifacts\docker-context"
 
+  if (Test-Path $dockerContext) {
+      Remove-Item -Recurse -Force $dockerContext
+  }
+
+  Copy-Item -Recurse -Destination $dockerContext\microsoft.netcore.app.ref -Path $binArtifacts\microsoft.netcore.app.ref
```

Comment:
NIT: can we swap order of -Path and -Destination? Or did you take the ordering here from somewhere else where we use it this way?
I was super confused since copy sans options is usually `source destination` argument order.